### PR TITLE
Feature/pro 903 add positions data to investments

### PR DESCRIPTION
**Pull request by kaushalrajbacancy**
- Added position data to account investments from zapper

## Description

- Detailed data was absent while fetching app holding of the users. We have added so that user can check all the meta details of its holding.

## Motivation and Context

-

## How Has This Been Tested?

- This has been tested locally through SDK example files.
-
-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by marie-fourier — APPROVED** [review]

**Review by vignesha22 — APPROVED** [review]

File: src/sdk/account/classes/account-investment-positions-info.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { BigNumber } from 'ethers';
+import { TransformBigNumber } from '../../common';
+
+export class AccountInvestmentPositionsInfo {
+    label: string;
```
Comment (ch4r10t33r) [inline src/sdk/account/classes/account-investment-positions-info.ts:5]: What do the fields label, metaType represent?

Comment (kaushalrajbacancy) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: It is a name

Comment (kaushalrajbacancy) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: The reason to keep both name and label as Zapper returns a different value 

```
 {
                "label": "Claimable AAVE",
                "balance": "0x2eb167f5f0e794",
                "tokens": 0.00023638325529935,
                "price": 55.6,
                "metaType": "claimable",
                "name": "AAVE",
                "logoURI": "https://storage.googleapis.com/zapper-fi-assets/tokens/ethereum/0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9.png"
              
}
```

Comment (kaushalrajbacancy) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: 13-Gwei-Zapper-Your-Home-to-Web3

Comment (ch4r10t33r) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: But others wont support this, no? Is this used anywhere within our dashboard or app?

Comment (kaushalrajbacancy) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: @ch4r10t33r In our app, we are not showing that label. We are sending multiple arrays in the API. So, if we try to display it like a zapper in the future, the label will be useful to distinguish

I would agree, other providers might not have. I can remove it if it is unnecessary for now. What is your thought? If yes, I will remove it from the API too.

Comment (ch4r10t33r) [reply src/sdk/account/classes/account-investment-positions-info.ts:5]: As this field is specific to zapper, I suggest we remove it as other providers might not support it.

Comment (kaushalrajbacancy) [reply src/sdk/account/classes/account-investment-positions-info.ts]: @ch4r10t33r Makes sense. Removed.

**Review by ch4r10t33r — APPROVED** [review]

**Pull request merged**